api/api.gradle.kts

```diff
@@ -10,5 +10,6 @@ dependencies {
     api(group = "com.google.guava", name = "guava", version = "21.0")
     api(group = "org.fxmisc.easybind", name = "easybind", version = "1.0.3")
     api(group = "org.controlsfx", name = "controlsfx", version = "8.40.11")
-    api(group = "edu.wpi.first.wpilib.networktables.java", name = "NetworkTables", version = "+", classifier = "desktop")
-}
\ No newline at end of file
+    api(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "+")
```

Comment:
@SamCarlberg and @nightpool 
Just curious, do you really want network tables as part of the public API of shuffleboard?
If not, you just want it to be an internal implementation detail then this should use the `implementation` configuration instead.